Origin: EmberBSD, AI-assisted.

gssapi 1.3.1 needs ffi >= 1.0.1 and gyoku 1.4.0 needs builder >= 2.1.2
at run time, but neither package depends on them, so RubyGems cannot
activate them (and winrm through them) unless something else happens to
install ffi and builder.  Declare both dependencies.
Not submitted upstream.

--- a/security/ruby-gssapi/Makefile
+++ b/security/ruby-gssapi/Makefile
@@ -1,12 +1,15 @@
 # $NetBSD: Makefile,v 1.2 2021/01/14 14:17:06 taca Exp $
 
 DISTNAME=	gssapi-1.3.1
+PKGREVISION=	1
 CATEGORIES=	security
 
 MAINTAINER=	pkgsrc-users@NetBSD.org
 HOMEPAGE=	https://github.com/zenchild/gssapi
 COMMENT=	FFI wrapper around the system GSSAPI library
 LICENSE=	mit
+
+DEPENDS+=	${RUBY_PKGPREFIX}-ffi>=1.0.1:../../devel/ruby-ffi
 
 USE_LANGUAGES=	# none
 
--- a/textproc/ruby-gyoku/Makefile
+++ b/textproc/ruby-gyoku/Makefile
@@ -1,12 +1,15 @@
 # $NetBSD: Makefile,v 1.2 2022/08/29 13:08:54 taca Exp $
 
 DISTNAME=	gyoku-1.4.0
+PKGREVISION=	1
 CATEGORIES=	textproc
 
 MAINTAINER=	pkgsrc-users@NetBSD.org
 HOMEPAGE=	https://github.com/savonrb/gyoku
 COMMENT=	Translates Ruby Hashes to XML
 LICENSE=	mit
+
+DEPENDS+=	${RUBY_PKGPREFIX}-builder>=2.1.2:../../textproc/ruby-builder
 
 USE_LANGUAGES=	# none
 
